build.gradle

```diff
@@ -54,8 +52,8 @@ compileJava {
 }
 
 compileJava9Java {
+	options.compilerArgs.addAll(['--release', '11'])
 	sourceCompatibility = 9
-	targetCompatibility = 9
```

Comment:
Not that I mind terribly, but why not leave it to be 9 compatible? Is there anything in particular that you worry about?

Reply:
Doesn't compile with --release 9 with Java 11. Guessing it wasn't officially support in 9 so it is excluded.

Reply:
Moved it completely to Java11. Let me know if you want me to revert it back to support 9, but it should not be a used JDK version today.

Reply:
That's fine then. It's not supported to run anything on JDK 9 and 10 in production today anyways.